tests/compute/subresources/pausing.go

```diff
@@ -273,28 +246,19 @@ var _ = compute.SIGDescribe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com
 			By("Trying to console into the VM")
 			_, err = virtClient.VirtualMachineInstance(vm.ObjectMeta.Namespace).SerialConsole(vm.ObjectMeta.Name, &kvcorev1.SerialConsoleOptions{ConnectionTimeout: 30 * time.Second})
 			Expect(err).ToNot(HaveOccurred())
-		})
-
-		It("[test_id:3084]should connect to vnc console", func() {
-			By("Pausing the VM")
-			err := virtClient.VirtualMachineInstance(vm.Namespace).Pause(context.Background(), vm.Name, &v1.PauseOptions{})
-			Expect(err).ToNot(HaveOccurred())
-			Eventually(matcher.ThisVM(vm), 30*time.Second, time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachinePaused))
 
 			By("Trying to vnc into the VM")
 			_, err = virtClient.VirtualMachineInstance(vm.ObjectMeta.Namespace).VNC(vm.ObjectMeta.Name)
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("[test_id:7673]should not pause with dry run", func() {
+		It("[test_id:7673][test_id:7674]should not pause and unpause with dry run", func() {
```

Comment:
Great merge but these should be moved imho if anything. I would probably reconsider if we can test dry-run as part of unit tests as it is only nice to have functionality.

Reply:
We can after https://github.com/kubevirt/kubevirt/pull/13052 will be merged